ext/Makefile

```diff
@@ -5,17 +5,27 @@ all: rawspeed
 
 rawspeed: Makefile $(BUILT)/rawspeed/librawspeed.a
 
+LDFLAGS = "-L../../../built/ext/rawspeed/src/external/zlib/build -lz"
+
 $(BUILT)/rawspeed/Makefile: Makefile
 ifneq ($(VKDT_USE_RAWSPEED),1)
 	mkdir -p $(BUILT)/rawspeed
 	touch $(BUILT)/rawspeed/Makefile
 else
 ifeq ($(RAWSPEED_PACKAGE_BUILD),0)
 	mkdir -p $(BUILT)/rawspeed
-	cd $(BUILT)/rawspeed; cmake ../../../ext/rawspeed -G "Unix Makefiles" -DBUILD_TESTING=0 -DCMAKE_BUILD_TYPE=release -DRAWSPEED_ENABLE_LTO=off -DRAWSPEED_ENABLE_DEBUG_INFO=off -DBUILD_BENCHMARKING=off -DBUILD_FUZZERS=off -DBUILD_TOOLS=off
+	cd $(BUILT)/rawspeed; LDFLAGS=$(LDFLAGS) PKG_CONFIG_PATH="/opt/homebrew/opt/zlib/lib/pkgconfig" cmake ../../../ext/rawspeed -G "Unix Makefiles" -DBUILD_TESTING=0 -DCMAKE_BUILD_TYPE=release -DRAWSPEED_ENABLE_LTO=off -DRAWSPEED_ENABLE_DEBUG_INFO=off -DBUILD_BENCHMARKING=off -DBUILD_FUZZERS=off -DBUILD_TOOLS=off \
+	  -DCMAKE_C_COMPILER=/opt/homebrew/opt/llvm/bin/clang -DCMAKE_CXX_COMPILER=/opt/homebrew/opt/llvm/bin/clang++ \
```

Comment:
does this not pick up the CC and CXX variables from the top level makefile?

Reply:
I had to do that because Apple clang does not have libomp. I will investigate a solution.
I forgot platform checks here sorry.